### PR TITLE
Add “No communication” card

### LINKED ISSUE #37: Add a "no communication" card to the dashboard
On the dashboard, under communication, add a card listing every planned upcoming talk without communication set up. Do not add an icon to the card

### PR DESCRIPTION
**Pull request by Cykelero**
Fixes #37

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -90,6 +90,15 @@ const Talks = () => {
 };
 
 const Communication = () => {
+    return (
```
Comment (zyhou) [inline pages/index.js]: This component is useless now. You can directly add this in `<Home />` component
ie
```js
<h2>Communication</h2>
    <Grid>
         <NextTweet />
         <NoCommunication />
    </Grid>
```

File: pages/index.js [hunk 2]
```diff
@@ -118,6 +127,32 @@ const Communication = () => {
...
+    if (isError) return <CardWithError title="No communication" onFailureRetry={retry} />;
+    if (isLoading || !plannedTalks) return <CardWithLoading title="No communication" />;
+
+    return (
+        <Card title="No communication">
+            <ul>
+                {plannedTalks
+                    .filter(plannedTalk => plannedTalk.tweets && plannedTalk.tweets.length)
+                    .map(plannedTalk => (
+                        <li key="{plannedTalk.name}">{plannedTalk.name}</li>
```
Comment (zyhou) [inline pages/index.js]:
```suggestion
                        <li key={plannedTalk.name}>{plannedTalk.name}</li>
```

**Review by zyhou — CHANGES_REQUESTED** [review]
Can you rebase?

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/**](https://aperoweb-git-add-no-communication.euregan.now.sh/) |
|:-:|
|<a href="https://aperoweb-git-add-no-communication.euregan.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/67ukIG8TJ_GrqYzWwcM666_rsi8_">(view full size)</a> |







Commit 5051da0d4d02b6099be0cde7f681fcf4773c79ad (https://aperoweb-git-add-no-communication.euregan.now.sh).

**Comment by Cykelero** [comment]
I've applied the changes you suggested; we should still check that the code works as expected, though.

**Review by zyhou — DISMISSED** [review]

**Comment by Cykelero** [comment]
The branch is now rebased on top of the latest mater commits.

**Review by zyhou — APPROVED** [review]

**Pull request merged**